src/vs/workbench/contrib/search/browser/searchWidget.ts

```diff
@@ -170,6 +171,7 @@ export class SearchWidget extends Widget {
 	public contextLinesInput!: InputBox;
 
 	private _notebookFilters: NotebookFindFilters;
+	private _toggleReplaceButtonListener: IDisposable | undefined;
```

Comment:
Perfect place to use `MutableDisposable`

Reply:
Fixed!